x/builder/client/cli/tx.go

```diff
@@ -0,0 +1,76 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/skip-mev/pob/x/builder/types"
+	"github.com/spf13/cobra"
+)
+
+// NewTxCmd returns a root CLI command handler for all x/builder transaction
+// commands.
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Builder transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		NewAuctionBidTx(),
+	)
+
+	return txCmd
+}
+
+func NewAuctionBidTx() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "auction-bid [bidder] [bid] [bundled_tx1,bundled_tx2,...,bundled_txN]",
```

Comment:
We need a way to specify the `Timeout` of an auction transaction. Is that something can be configured directly on CLI as is? I'm not too familiar with what flags are available for CLI txs.

Reply:
`flags.AddTxFlagsToCmd(cmd)` handles all global flag stuff for us. I added an explicit check that it must be positive.